examples/client.rs

```diff
@@ -74,3 +108,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     Ok(())
 }
+
+struct InteropVerifier(Arc<Mutex<bool>>);
+impl rustls::ServerCertVerifier for InteropVerifier {
+    fn verify_server_cert(
+        &self,
+        _roots: &rustls::RootCertStore,
+        _presented_certs: &[rustls::Certificate],
+        _dns_name: webpki::DNSNameRef,
+        _ocsp_response: &[u8],
+    ) -> std::result::Result<rustls::ServerCertVerified, rustls::TLSError> {
+        *self.0.lock().unwrap() = true;
```

Comment:
Curious, what does setting this boolean do? It doesn't seem it's checked anywhere else... Could this just be an empty struct?

Reply:
It completely useless. It's from a not-edited copy paste from Quinn's interop code.
I removed it.

Thanks